DEPS.py

```diff
@@ -4,7 +4,7 @@
 
 deps = {
     'ACT': {
-        'url': 'https://github.com/EQAditu/AdvancedCombatTracker/releases/download/3.4.7.268/ACTv3.zip',
+        'url': 'https://github.com/EQAditu/AdvancedCombatTracker/releases/download/3.4.9.271/ACTv3.zip',
         'dest': 'Thirdparty/ACT',
         'strip': 0,
         'hash': ['sha256', 'adf13a38d0938ce90f8e674f8365b227d933b91636ddf72b26c85702f6e3b808'],
```

Comment:
Does this hash need to be updated as well, then?

Reply:
Yeah, I can update the hash, but it appears as though either the `tools/fetch_deps.py` script either isn't checking the hash or isn't warning or erroring on a bad hash.

As an aside, running that script from a bash terminal (which is my only python install right now) mangles the FFXIV SDK paths and I have no where near the level of python knowledge to actually fix the issue.

```bash
ls Thirdparty/FFXIV_ACT_Plugin 
 FFXIV_ACT_Plugin.dll         'SDK\FFXIV_ACT_Plugin.Common.dll'  'SDK\FFXIV_ACT_Plugin.LogFile.dll'  'SDK\FFXIV_ACT_Plugin.Network.dll'  'SDK\FFXIV_ACT_Plugin.Parse.dll'
 FFXIV_ACT_Plugin_Korean.dll  'SDK\FFXIV_ACT_Plugin.Config.dll'  'SDK\FFXIV_ACT_Plugin.Memory.dll'   'SDK\FFXIV_ACT_Plugin.Overlay.dll'  'SDK\FFXIV_ACT_Plugin.Resource.dll'
```

Reply:
Oof.  If you have a chance, could you file issues for both of those and maybe somebody (me?) can remember to take a look?

Reply:
Hmm, even more issues with that hashing logic. Apparently if I'm reading it right, it only hashes the first 16kb of the file. And with that context (only the first 16kb of the file) the hashes of the two versions are identical:

```bash
$ curl https://github.com/EQAditu/AdvancedCombatTracker/releases/download/3.4.9.271/ACTv3.zip -L -o ACTv3.zip --silent
$ head -c $((1024 * 16)) ACTv3.zip | sha256sum
adf13a38d0938ce90f8e674f8365b227d933b91636ddf72b26c85702f6e3b808  -
$ curl https://github.com/EQAditu/AdvancedCombatTracker/releases/download/3.4.7.268/ACTv3.zip -L -o ACTv3.zip --silent
$ head -c $((1024 * 16)) ACTv3.zip | sha256sum
adf13a38d0938ce90f8e674f8365b227d933b91636ddf72b26c85702f6e3b808  -
```

Reference:

https://github.com/OverlayPlugin/OverlayPlugin/blob/9a63992c20b54d09d0b3842af0de5c421e4766d8/tools/fetch_deps.py#L98-L110

Reply:
Oh wow.  Ok, I guess it's ok then!